backend/app/tests/crud/test_patient.py

```diff
@@ -0,0 +1,40 @@
+
+import uuid
+from app.models import PatientCreate, PatientUpdate, UserCreate, PatientsPublic
+from app.tests.utils.utils import random_email, random_lower_string, random_phone_number, rendom_patient
+from sqlmodel import Session
+from app import crud
+from datetime import datetime
+
+
+
+def test_patient(db: Session) -> None:
+    user_in = UserCreate(email = random_email(), password = random_lower_string() )
+    user = crud.create_user(session=db, user_create=user_in)
+    patient = crud.create_patient(session=db, patient_create=rendom_patient() ,owner_id=user.id)
+    # create patient test
+    assert patient.owner_id == user.id
+    patient1 = crud.create_patient(session=db, patient_create=rendom_patient() ,owner_id=user.id)
+    for num in range(200):
+        patient_rundom = crud.create_patient(session=db, patient_create=rendom_patient() ,owner_id=user.id)
+        assert patient_rundom.id != patient1.id
+    patients_found = crud.get_caregiver_patients(session=db, caregiver=user, skip=0, limit=100)
+    assert type(patients_found) == PatientsPublic
+    assert patients_found.count == 202
```

Comment:
This is cool.
You should also verify that `skip` and `limit` work, by counting the number of patients you got back for limit, and by querying again with skip = 1 and see if p[0] == p_with_skip[1]